src/database/index.tsx

```diff
@@ -433,10 +433,31 @@ export const deleteFacility = async (facilityId: string): Promise<void> => {
 
     if (!baselineScenarioRef) return;
 
-    await baselineScenarioRef
+    // Delete all of the modelVersions associated with a facility.  Technically,
+    // this is not the recommended approach for the web*, but we should be ok
+    // while our datasets are small.  In the future, we can address this issue
+    // if it becomes problematic**.
+    //
+    // * https://firebase.google.com/docs/firestore/manage-data/delete-data#web_2
+    // ** https://github.com/Recidiviz/covid19-dashboard/issues/191
```

Comment:
Would you mind filing a tech debt ticket to migrate deletion to a Cloud Function, as [described in their docs](https://firebase.google.com/docs/firestore/solutions/delete-collections)? It doesn't look terribly challenging, so I think having an issue dedicated to it in the future seems sensible.

Reply:
[The ticket](https://github.com/Recidiviz/covid19-dashboard/issues/191) mentioned in the comment covers this work and references the specific document URL you mentioned.  I updated it to have the "Tech Debt" label though.

Reply:
I'll also update the copy in the ticket to be more up-to-date with our current reality (i.e. handling the deletion client-side and migrating that to a cloud function).

Reply:
Ah yes of course. Thank you!